src/screens/SendToken/SendTokenPinConfirmScreen.js

```diff
@@ -132,11 +128,12 @@ class SendTokenPinConfirmScreen extends React.Component<Props, State> {
   };
 
   navigateToTransactionState = (params: ?Object) => {
-    const { navigation } = this.props;
+    const { navigation, transactionNotificationPayload } = this.props;
     const { transactionPayload } = this.state;
     const transactionType = navigation.getParam('transactionType', '');
 
-    navigation.navigate(SEND_TOKEN_TRANSACTION, { ...params, transactionPayload, transactionType });
+    transactionNotificationPayload({ ...transactionPayload, ...params, transactionType });
```

Comment:
See comment above re: naming, the function name here doesn't tell me what is happening

Reply:
I updated name here